### PR TITLE
Issue94

### PR DESCRIPTION
**Pull request by wsbaser**

### CODE REVIEW

File: plugin/src/main/java/org/websync/websession/psimodels/PsiWebSession.java [hunk 1]
```diff
@@ -22,4 +22,13 @@ public PsiWebSession(Collection<PsiWebsite> websites, Collection<PsiComponentTyp
         this.pageTypes = pages.stream()
                 .collect(Collectors.toMap(PsiPageType::getId, page -> page));
     }
-}
+
+    public void updateComponentInstanceInType(String id, String oldName, String newName) {
```
Comment (wsbaser) [inline plugin/src/main/java/org/websync/websession/psimodels/PsiWebSession.java:26]: do we still need these methods?

File: plugin/src/main/java/org/websync/WebSyncService.java [hunk 2]
```diff
@@ -96,4 +96,20 @@ public void removeProject(Project project) {
         this.provider.removeProject(project);
     }
 
+    public void updateComponentInstance(String className, String oldFieldName, String newFieldName) {
```
Comment (wsbaser) [inline plugin/src/main/java/org/websync/WebSyncService.java:99]: not sure that this class has this responsibility

File: plugin/src/main/java/org/websync/websession/psimodels/PsiWebSession.java (hunk 1 above)
Comment (chmutov) [reply plugin/src/main/java/org/websync/websession/psimodels/PsiWebSession.java:26]: no, will remove it

File: plugin/src/main/java/org/websync/WebSyncService.java (hunk 2 above)
Comment (chmutov) [reply plugin/src/main/java/org/websync/WebSyncService.java:99]: In general, I agree with you. But now it's the best place for it. The next step is redesign the org.websync.websession.psimodels package and provide the correct way to store and update PSI structures in the IDE.

**Pull request merged**